src/communication/dryrun_global_policy.hpp

```diff
@@ -74,6 +74,13 @@ struct dryrun_global_policy {
         return size()*value;
     }
 
+    template <typename T>
```

Comment:
it has been added to the `global_policy` interface. Each of the policy implementations (dry run, serial and mpi) expose a similar call.
We don't have any documentation for the other calls in the API. The whole thing should be documented properly.
I will add a comment that explains what this call does.